data/deprecatedIdentifiers/deprecatedReactions.tsv

```diff
@@ -1,301 +1,346 @@
-"MAR00789"	""	"FAOXC183803m"	""	""	""	"FAOXC183803m"	"MNXR99325"	""	""	""	0	""	""	"FAOXC183803m"
-"MAR03426"	""	"HMR_3426"	""	""	""	"HMR_3426"	""	"HMR_3426"	"RCR12671"	""	0	""	""	"HMR_3426"
-"MAR05019"	""	"FAOXC10C10OHm"	""	""	""	"FAOXC10C10OHm"	""	""	""	""	0	""	""	"FAOXC10C10OHm"
-"MAR04967"	""	"FAOXC101C8m"	""	""	""	"FAOXC101C8m"	""	""	""	""	0	""	""	"FAOXC101C8m"
-"MAR05024"	""	"FAOXC10080m"	""	""	""	"FAOXC10C8m"	"MNXR99262"	""	""	""	0	""	""	"FAOXC10C8m"
-"MAR03322"	""	""	"RE3627X"	""	""	"HMR_3322"	"MNXR124610"	"HMR_3322"	"RCR14521"	""	0	""	""	"HMR_3322"
-"MAR03296"	""	""	"RE3627M"	""	"R-HSA-109343"	"HMR_3296"	"MNXR124610"	"HMR_3296"	"RCR12779"	""	0	""	""	"HMR_3296"
-"MAR04005"	""	""	""	""	""	"RE3627C"	""	""	""	""	0	""	""	"RE3627C"
-"MAR05187"	""	"FAOXC15NADx"	""	""	""	"FAOXC15NADx"	"MNXR99293"	""	""	""	0	""	""	"FAOXC15NADx"
-"MAR05186"	""	"FAOXC15NADPx"	""	""	""	"FAOXC15NADPx"	"MNXR99292"	""	""	""	0	""	""	"FAOXC15NADPx"
-"MAR05358"	""	"FAOXC183C163Gm"	""	""	""	"FAOXC183C163Gm"	""	""	""	""	0	""	""	"FAOXC183C163Gm"
-"MAR05195"	""	"FAOXC163C164Gm"	""	""	""	"FAOXC163C164Gm;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC163C164Gm;HMR_3422;MAR03422"
-"MAR05302"	""	"FAOXC164GC163m"	""	""	""	"FAOXC164GC163m;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC164GC163m;HMR_3422;MAR03422"
-"MAR05229"	""	"FAOXC163Gm"	""	""	""	"FAOXC163Gm;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC163Gm;HMR_3422;MAR03422"
-"MAR05208"	""	"FAOXC163GC142m"	""	""	""	"FAOXC163GC142m;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC163GC142m;HMR_3422;MAR03422"
-"MAR05356"	""	"FAOXC182C162m"	""	""	""	"FAOXC182C162m"	""	""	""	""	0	""	""	"FAOXC182C162m"
-"MAR05191"	""	"FAOXC162C142m"	""	""	""	"FAOXC162C142m"	""	""	""	""	0	""	""	"FAOXC162C142m"
-"MAR05123"	""	"FAOXC142C122m"	""	""	""	"FAOXC142C122m;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC142C122m;HMR_3422;MAR03422"
-"MAR05075"	""	"FAOXC122m"	""	""	""	"FAOXC122m;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC122m;HMR_3422;MAR03422"
-"MAR05072"	""	"FAOXC122C101m"	""	""	""	"FAOXC122C101m;HMR_3422"	""	"HMR_3422"	"RCR12667"	""	0	""	""	"FAOXC122C101m;HMR_3422;MAR03422"
-"MAR00965"	""	"FAOXC2252053m"	""	""	""	"FAOXC2252053m"	"MNXR99352"	""	""	""	0	""	""	"FAOXC2252053m"
-"MAR04774"	"R03237"	""	"R03237C"	"r0610"	""	"r0610"	"MNXR105355;MNXR108039"	"HMR_4774"	"RCR11046"	""	0	""	""	"HMR_4774"
-"MAR04775"	"R03239"	""	"R03239C"	"r0611"	""	"r0611"	"MNXR105356;MNXR108041"	"HMR_4775"	"RCR14197"	""	0	""	""	"HMR_4775"
-"MAR01788"	""	"THMMPtm"	""	""	""	"THMMPtm"	"MNXR104822"	""	""	""	0	""	""	"THMMPtm"
-"MAR01789"	""	"THMPPtm"	""	""	""	"THMPPtm"	"MNXR104824"	""	""	""	0	""	""	"THMPPtm"
-"MAR04205"	""	""	"RT0084"	""	""	"HMR_4205"	""	"HMR_4205"	"RCR20258"	""	0	""	""	"HMR_4205"
-"MAR08745"	""	"THMMPtm"	""	""	""	""	"MNXR104822"	"HMR_8745"	"RCR20669"	""	0	""	""	"HMR_8745"
-"MAR08747"	""	"THMPPtm"	""	""	""	""	"MNXR104824"	"HMR_8747"	"RCR20670"	""	0	""	""	"HMR_8747"
+MAR01459		HSD17B7r				HSD17B7r	MNXR100715				0			HSD17B7r
```

Comment:
`metabolites.tsv` in the `model/` folder is using quotation marks, even on the header line, but `reactions.tsv` does not.
In this file quotesa are removed.
For consistency, the same style should be used in both instances.